templates/Common/DSMELB.template

```diff
@@ -1,260 +1,118 @@
-            }
-        },
-        "ELSBSourceSecurityGroupOnwerID": {
-            "Value": {
-                "Fn::GetAtt": [
-                    "DSMELB",
-                    "SourceSecurityGroup.OwnerAlias"
-                ]
-            }
-        },
-        "DSIELB": {
-            "Value": {
-                "Ref": "DSMELB"
-            }
-        },
-        "PrivateDSIELB": {
-            "Value": {
-                "Ref": "PrivateDSMELB"
-            }
-        },
-        "PrivateELBFQDN": {
-            "Value": {
-                "Fn::GetAtt": [
-                    "PrivateDSMELB",
-                    "DNSName"
-                ]
-            }
-        },
-        "PrivateELBSourceSecurityGroup": {
-            "Value": {
-                "Fn::GetAtt": [
-                    "ELBSG",
-                    "Outputs.ELBSG"
-                ]
-            }
-        },
-        "PrivateELSBSourceSecurityGroupOnwerID": {
-            "Value": {
-                "Fn::GetAtt": [
-                    "PrivateDSMELB",
-                    "SourceSecurityGroup.OwnerAlias"
-                ]
-            }
-        },
-        "PublicELBCanonicalHostedZoneNameID": {
-            "Value": {
-                "Fn::GetAtt": [
-                    "DSMELB",
-                    "CanonicalHostedZoneNameID"
-                ]
-            }
-        },
-        "PublicELBDNSName": {
-            "Value": {
-                "Fn::GetAtt": [
-                    "DSMELB",
-                    "DNSName"
-                ]
-            }
-        }
-    }
-}
\ No newline at end of file
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  v5.3: Deploys Elastic Load Balancers and Security Groups for Deep Security
+  Manager.
+Parameters:
+  AWSIVPC:
+    Description: Existing VPC to deploy Deep Security Manager
+    Type: 'AWS::EC2::VPC::Id'
+    MinLength: '1'
+    MaxLength: '255'
+    AllowedPattern: '[-_a-zA-Z0-9]*'
+  DSIPHeartbeatPort:
+    Description: >-
+      The heartbeat port used by Deep Security Agents and appliances to
+      communicate with the Deep Security Manager.
+    Type: Number
+    MinValue: 0
+    MaxValue: 65535
+    Default: '4120'
+    ConstraintDescription: Must be a valid TCP port.
+  DSIPGUIPort:
+    Description: The Deep Security Manager application and GUI port.
+    Type: Number
+    MinValue: 0
+    MaxValue: 65535
+    Default: '4119'
+    ConstraintDescription: Must be a valid TCP port.
+  DSISubnetID:
+    Description: >-
+      Existing Subnet for Deep Security Manager. Must be a public subnet
+      contained the in VPC chosen above.
+    Type: String
+    MinLength: '1'
+    MaxLength: '255'
+    AllowedPattern: '[-_a-zA-Z0-9]*'
+    ConstraintDescription: Subnet ID must exist in the chosen VPC
+  DSELBPosture:
+    Description: Use internal or internet-facing ELB
+    Type: String
+    AllowedValues:
+      - Internet-facing
+      - Internal
+    Default: Internet-facing
+  CfnUrlPrefix:
```

Comment:
We don't recommend using a parameter like this, but instead using QSS3BucketName and QSS3KeyPrefix and dynamically building a virtual-hosted S3 url from it via Fn::Sub similar to https://github.com/aws-quickstart/quickstart-linux-bastion/blob/develop/templates/linux-bastion-master.template#L310